chromium_src/chrome/browser/chrome_content_browser_client.cc

```diff
@@ -6,13 +6,23 @@
 #include "build/build_config.h"  // For OS_MAC
 #include "chrome/browser/search/search.h"
 #include "content/public/browser/browser_url_handler.h"
+#include "extensions/buildflags/buildflags.h"
 
 #if defined(OS_MAC)
 #include "brave/browser/brave_browser_main_parts_mac.h"
 #undef ChromeBrowserMainPartsMac
 #define ChromeBrowserMainPartsMac BraveBrowserMainPartsMac
 #endif
 
+#if !BUILDFLAG(ENABLE_EXTENSIONS)
+#include "ui/gfx/image/image_skia.h"
```

Comment:
This is not right, `ui/gfx/image/image_skia.h` is not related to the enable_extensions flag. Buildflags and gn guards should always match for header includes regardless of whether or not the class only appears inside a different buildflag in the cpp code

Reply:
Fixed in 7bbc3fc